Composer/packages/extensions/visual-designer/src/widgets/SwitchConditionWidget.tsx

```diff
@@ -114,7 +114,9 @@ export const SwitchConditionWidget: FunctionComponent<SwitchConditionWidgetProps
           />
         </OffsetContainer>
       ))}
-      <SVGContainer>{Array.isArray(edges) ? edges.map(x => renderEdge(x)) : null}</SVGContainer>
+      <SVGContainer width={boundary.width} height={boundary.height}>
+        {Array.isArray(edges) ? edges.map(x => renderEdge(x)) : null}
```

Comment:
Looks like this logic is repeated a few times. Can we compose this logic inside the SVGContainer component itself?

Reply:
It would be better if we have a `<FlowEdges />` component

Reply:
@srinaath I resolved it by adding a new component `<FlowEdges edges={edges} />`